### PR TITLE
AMP-92829 Add event filtering note for DWH and CS destinations

### PR DESCRIPTION
**Pull request by LeontiBrechko**
# Amplitude Developer Docs PR

## Description

Adds a note about event filtering support for Data Warehouse and Cloud Storage destinations

## Deadline
March 1, 2024

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/amazon-s3.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To set up the Amazon S3 integration, follow these steps:
 
 1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
 2. In the Warehouse Destination section, click **Amazon S3**.
-3. Choose which data you'd like to include in this export: *Export events ingested today and moving forward*, *Export all merged Amplitude IDs*, or export both.
+3. Choose which data you'd like to include in this export: *Export events ingested today and moving forward*, *Export all merged Amplitude IDs*, or export both. Also, you can specify event filtering.
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:34]: Do we want to word this from the angle of what value to the customer?
For example, 
For events, you can also specify filtering conditions to only export events that meets these conditions.

**Review by joez2 — APPROVED** [review]
Thanks for the doc update! Just a small suggestion on the wording

File: docs/data/destinations/amazon-s3.md (hunk 1 above)
Comment (markzegarelli) [reply docs/data/destinations/amazon-s3.md:34]: I agree, I think a little more context here will be helpful.

Comment (LeontiBrechko) [reply docs/data/destinations/amazon-s3.md:34]: any proposition on the wording/structure here?😅 don't want to bloat the docs with extra screenshots/descriptions for specific edge-cases

Comment (LeontiBrechko) [reply docs/data/destinations/amazon-s3.md:34]: Or did you mean `For events, you can also specify filtering conditions to only export events that meet these conditions.` as a specific example of the wording? if yes, this specific wording works for me :)

Comment (LeontiBrechko) [reply docs/data/destinations/amazon-s3.md:34]: or `For events, you can also specify filtering conditions to only export events that meet certain conditions.`

Comment (joez2) [reply docs/data/destinations/amazon-s3.md:34]: Yeah, this is good.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**